ui/components/app/add-network/add-network.js

```diff
@@ -235,9 +237,10 @@ const AddNetwork = () => {
                         }
                         trigger="mouseenter"
                       >
-                        <i
-                          className="fa fa-exclamation-triangle add-network__warning-icon"
-                          title={t('warning')}
+                        <Icon
+                          name={ICON_NAMES.DANGER}
+                          color={Color.iconMuted}
+                          size={ICON_SIZES.SM}
```

Comment:
I think we should not remove the className from here
```suggestion
                        <Icon
                          className="add-network__warning-icon"
                          name={ICON_NAMES.DANGER}
                          color={Color.iconMuted}
                          size={ICON_SIZES.SM}
```